pythonforandroid/recipes/matplotlib/mpl_android_fixes.patch

```diff
@@ -4,26 +4,26 @@ index fc82d5d..2067db0 100644
 +++ b/setupext.py
 @@ -1004,10 +1004,10 @@ class Numpy(SetupPackage):
          ext.define_macros.append(('__STDC_FORMAT_MACROS', 1))
- 
+
      def get_setup_requires(self):
 -        return ['numpy>=1.10.0']
-+        return ['numpy==1.15.1']  # to match p4a's target version
- 
++        return ['numpy==1.16.4']  # to match p4a's target version
+
      def get_install_requires(self):
 -        return ['numpy>=1.10.0']
-+        return ['numpy==1.15.1']  # to match p4a's target version
- 
- 
++        return ['numpy==1.16.4']  # to match p4a's target version
```

Comment:
Just wondering if this patch is really required? Prior the patch it's with `numpy>=1.10.0` which doesn't conflict with `numpy==1.16.4` correct?

Reply:
Yes, it's required, we need matplotlib to install and build against the same version of numpy that p4a builds or there can be incompatibilities. It would be better to get matplotlib to actually use the numpy headers from the p4a build, but wasn't necessary to get the recipe working so I didn't bother.

Reply:
OK thanks for the info :+1: